.github/workflows/pull-build-test.yml

```
@@ -0,0 +1,44 @@
+name: pull-build-test
+
+on:
+  pull_request:
+    branches:
+      - master
+      - feature/**
+      - release-fts-*
+    branches-ignore:
```

actionlint (1.7.12)

[error] both "branches" and "branches-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)